assets/js/scripts.js

```diff
@@ -450,18 +450,17 @@ $('.collection-reset').on('click', function (e) {
 
     if (inventory[value.text])
       inventory[value.text].isCollected = false;
-
+    
     value.isCollected = false;
     value.canCollect = true;
 
-    if (value.subdata) {
-      Inventory.changeMarkerAmount(value.subdata, -1);
-      $(this).removeClass('disabled');
-    }
-    else {
-      Inventory.changeMarkerAmount(value.text, -1);
-      $(this).removeClass('disabled');
-    }
+    // .changeMarkerAmount() must run to check whether to remove "disabled" class
+    if (value.subdata) 
+      Inventory.changeMarkerAmount(value.subdata, (Inventory.resetButtonChangingInventory ? -1 : 0));
+    else
+      Inventory.changeMarkerAmount(value.text, (Inventory.resetButtonChangingInventory ? -1 : 0));

```

Comment:
Surround with `if (Inventory.resetButtonChangingInventory)` so it can be removed from `changeMarkerAmount`. ChangeMarkerAmount(0) doesn't do anything.

Reply:
here you are wrong
this doesn't work:
```
if (Inventory.resetButtonChangingInventory){
    if (value.subdata) 
        Inventory.changeMarkerAmount(value.subdata, -1);
    else
        Inventory.changeMarkerAmount(value.text, -1);
}
```
i am not sure why, that was my first idea, but changeMarkerAmount() must run at least once

Reply:
That's an issue with changeMarkerAmount, I should look into that.